gapis/api/vulkan/extensions/khr_descriptor_update_template.api

```diff
@@ -124,7 +123,10 @@ sub ref!DescriptorUpdateTemplateObject createDescriptorUpdateTemplate(
     if !(device in Devices) { vkErrorInvalidDevice(device) }
     if pCreateInfo == null { vkErrorNullPointer("pDescriptorUpdateTemplate") }
     info := pCreateInfo[0]
-    if !(info.descriptorSetLayout in DescriptorSetLayouts) { vkErrorInvalidDescriptorSetLayout(info.descriptorSetLayout) }
+    if !(info.descriptorSetLayout in DescriptorSetLayouts) {
+        vkErrorInvalidDescriptorSetLayout(info.descriptorSetLayout)
+        abort
```

Comment:
I am not sure how to handle the error here, as a return value is expected? Should we return nil, and if so how do we express this?

Reply:
@AWoloszyn a friendly ping :)

Reply:
You can make a nil object, but it is a bit awkward.

```
class NilDescriptorUpdateTemplateObject {
    ref!DescriptorUpdateTemplateObject v;
}

....
....
obj := NilDescriptorUpdateTemplateObject().v
if (info.descriptorSetLayout in DescriptorSetLayouts) {
  obj = .....
}
return obj
```

Reply:
Thanks, I ended up opting for:
```
...
obj := new!DescriptorUpdateTemplateObject()
if (check error) {
  // report error
} else {
  obj.Device = device
  // etc...
}
return obj
```